BraintreeModels/src/main/java/com/braintreepayments/api/InvalidToken.java

```diff
@@ -0,0 +1,25 @@
+package com.braintreepayments.api;
+
+class InvalidToken extends Authorization {
```

Comment:
```suggestion
class InvalidAuthorization extends Authorization {
```

Take it or leave it comment - renaming this since we don't really use `token` to mean `auth` consistently.

Reply:
We renamed this to `InvalidAuthorization` 👍